Project/DAQ_System/Program/Src/Sensor/Accelerometer/LSM6DSOXAccelerometer.hpp

```diff
@@ -12,20 +12,39 @@
 #ifndef ACCELEROMETER_LSM6DSOX_H
 #define ACCELEROMETER_LSM6DSOX_H
 
-
-// DFR Custom Dependancies
-#include "iaccelerometer.hpp"
-
 // ST HAL Dependencies
 #include "i2c.h"
 
+//  DFR Custom Dependencies
+#include "iaccelerometer.hpp"
+
 #define ACC_ADDRESS  0xd4
 
 // Register Addresses
 
-#define CTRL1_XL          0x10  // CTRL_REG1_A
+#define CTRL1_XL          0x10
+
+#define OUTX_L_A         0x28
+
+enum ODR{
+	ODR12_5,
+	ODR26,
+	ODR52,
+	ODR104,
+	ODR208,
+	ODR416,
+	ODR833,
+	ODR1_66K,
+	ODR3_33K,
+	ODR6_66K
+};
 
-#define OUT_X_L_A         0x28
+enum FSR{
+	FSR2g = 10,
```

Comment:
I saw something like this in the gyroscope code, but why is only this assigned with a value?

Reply:
the sequential enumerations auto-increment from the first initialization. So if FSR2g is assigned with 10 then FSR4g  = 11, FSR8G = 12, FSR16G = 13.